### PR TITLE
MTV-1680: providers fix

### PR DESCRIPTION
**Pull request by anarnold97**
### JIRA

* [MTV-1680](https://issues.redhat.com/browse/MTV-1680)

### Preview

* [Providers](https://anarnold97.github.io/MTA-Preview/MTV/provider-fix.html#adding-source-providers)

### CODE REVIEW

File: documentation/modules/rn-27-resolved-issues.adoc [hunk 1]
```diff
@@ -20,7 +20,7 @@ In earlier releases of {project-short}, having a large number of virtual machine
 
 .VM selection disappears when selecting multiple VMs in the Migration Plan
 
-In earlier releases of {project-short}, *VM selection* checkbox disappeared after selecting multiple VMs in the Migration Plan. This issue has been resolved in {project-short} 2.7.3. link:https://issues.redhat.com/browse/MTV-1546[(MTV-1546)]
+In earlier releases of {project-short}, the *VM selection* checkbox disappeared after selecting multiple VMs in the Migration Plan. This issue has been resolved in {project-short} 2.7.3. link:https://issues.redhat.com/browse/MTV-1546[(MTV-1546)]
```
Comment (anarnold97) [inline documentation/modules/rn-27-resolved-issues.adoc:23]:
```suggestion
In earlier releases of {project-short}, *VM selection* checkbox disappeared after selecting multiple VMs in the Migration Plan. This issue has been resolved in {project-short} 2.7.3. link:https://issues.redhat.com/browse/MTV-1546[(MTV-1546)]
```

File: documentation/modules/adding-source-provider.adoc [hunk 2]
```diff
@@ -39,7 +39,7 @@ You can add an {osp} source provider by using the {ocp} web console.
 
 [IMPORTANT]
 ====
-When you migrate an image-based VM from an {osp} provider, a snapshot is created for the image that is attached to the source VM and the data from the snapshot is copied over to the target VM. This means that the target VM will have the same state as that of the source VM at the time the snapshot was created.
+When you migrate an image-based VM from an {osp} provider, a snapshot is created for the image that is attached to the source VM and the data from the snapshot is copied over to the target VM. This means that the target VM will have the same state as that of the source VM at the time the snapadding-source-providershot was created.
```
Comment (anarnold97) [inline documentation/modules/adding-source-provider.adoc:42]:
```suggestion
When you migrate an image-based VM from an {osp} provider, a snapshot is created for the image that is attached to the source VM and the data from the snapshot is copied over to the target VM. This means that the target VM will have the same state as that of the source VM at the time the snapshot was created.
```

File: documentation/modules/adding-source-provider.adoc [hunk 3]
```diff
@@ -82,19 +82,23 @@ ifdef::vmware[]
 . Click *vSphere*.
 . Specify the following fields:
 
++
 *Provider Details*
```
Comment (RichardHoch) [inline documentation/modules/adding-source-provider.adoc:87]: Should be *Provider details* -- It's correct in the  UI.

File: documentation/modules/adding-source-provider.adoc [hunk 4]
```diff
@@ -82,19 +82,23 @@ ifdef::vmware[]
...
```
Comment (anarnold97) [inline documentation/modules/adding-source-provider.adoc:86]:
```suggestion
*Provider details*
```

**Pull request merged**